executor/runtime/docker/docker.go

```diff
@@ -991,7 +991,12 @@ func (r *DockerRuntime) Prepare(ctx context.Context) (err error) { // nolint: go
 		bindMounts = append(bindMounts, getKernelBindMounts()...)
 	}
 	pod, podLock := r.c.Pod()
+	mainContainerBindMounts, err := r.getBindMountsForContainer(pod.Spec.Containers[0], *pod)
+	if err != nil {
+		return err
```

Comment:
podLock.Unlock() here, or just move it above the err check?

Reply:
good idea